src/hotspot/share/gc/parallel/parallel_globals.hpp

```diff
@@ -83,7 +83,7 @@
   product(bool, UseIOPrioritySizePolicy, false,                             \
           "eagerly decrease heap when io wait is high")                     \
                                                                             \
-  product(float, IOPrioritySizePolicyEdenScale, 8.0,                        \
+  product(uint, IOPrioritySizePolicyEdenScale, 8,                           \
```

Comment:
Why does using `float` cause the build failure on aarch64?
Have you tried using `double`?

Reply:
Since this is an emergency, I haven't done much research. Build logs says
`11:32:59 ./src/hotspot/share/runtime/flags/jvmFlagConstraintList.cpp: In static member function 'static void JVMFlagConstraintList::init()':
11:32:59 ./src/hotspot/share/runtime/flags/jvmFlagConstraintList.cpp:244:70: error: 'emit_constraint_float' was not declared in this scope
`

Reply:
emit_constraint_float was not defined, I guess people don't use float arguments?

Reply:
Buy why it's okay on x64?